modules/virt-example-ethernet-nncp.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * virt/node_network/virt-updating-node-network-config.adoc
+
+[id="virt-example-ethernet-nncp_{context}"]
+= Example: Ethernet interface NodeNetworkConfigurationPolicy
+
+Configure an ethernet interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster.
```

Comment:
s/ethernet/Ethernet for all instances of `ethernet` outside of YAMLs (ISG p 327)

Reply:
Great catch, that passed me by completely